HISTORY.rst

```diff
@@ -3,6 +3,62 @@
 Release History
 ---------------
 
+2.0.0 (2023-04-15) Wellton
+++++++++++++++++++++++++++
+
+**Special notes**
+
+Mesa 2.0 includes several breaking changes that provide significant improvements to Mesa.
+
+**Breaking Changes:**
+
+* space: change `coord_iter` to return `(content,(x,y))` instead of `(content, x,y)`; this reduces known errors of scheduler to grid mismatch #1566, #1723
+* space: change NetworkGrid `get_neighbors` to `get_neighborhood`; improves performance #1542
+* space: raise exception when pos is out of bounds in `Grid.get_neighborhood` #1524
+* space: remove deprecations (#1520, #1687, #1688):
+    * `find_empty()`: convert this to `move_to_empty()`
+    * `num_agents`: removed parameter from `move_to_empty()`
+    * `position_agent()`: convert this to `place_agent`
+    * `neighbor_iter()`: convert this to `iter_neighborhood()`
+* batchrunner: remove deprecations #1627
+    * `class BatchRunner` and `class BatchRunnerMP`: convert these to `batch_run()`
+* visualization: easier visualization creation #1693
+    * `UserSettableParameter(['number', 'slider','checkbox', 'choice', `StaticText`]): convert to `NumberInput`,`Slider`, `CheckBox`, `Choice`, `StaticText`
+
+
+**New Features:**
+
+* datacollector: can now handle data collection by agent type #1419, #1702
+* time: allows for model level `StageActivation` #1709
+* visualization: `ChartModule` can have dynamically named properties #1685
+* visualization: improved stop server to end visualizations #1646
+* *experimental* python front end option: integrated the initial prototype of the pure python front end option #1698, #1726
```

Comment:
This needs to be emphasized in the early section of the release note. Anyone reading the release note would have missed this sentence very easily. You could say that this is the main change of 2.0, other than the breaking changes.

Also, needs to say early on that we are switching to the pure Python viz as the default, if it goes well.